packages/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.ts

```diff
@@ -83,14 +83,14 @@ export const usePresenceBadgeStyles_unstable = (state: PresenceBadgeState): Pres
   state.root.className = mergeClasses(
     presenceBadgeClassName,
     styles.root,
-    (state.status === 'busy' || state.status === 'doNotDisturb') && styles.statusBusy,
+    ['busy', 'doNotDisturb', 'unknown'].includes(state.status) && styles.statusBusy,
     state.status === 'away' && styles.statusAway,
     state.status === 'available' && styles.statusAvailable,
     state.status === 'offline' && styles.statusOffline,
     state.status === 'outOfOffice' && styles.statusOutOfOffice,
     state.outOfOffice && styles.outOfOffice,
     state.outOfOffice && state.status === 'available' && styles.outOfOfficeAvailable,
-    state.outOfOffice && (state.status === 'busy' || state.status === 'doNotDisturb') && styles.outOfOfficeBusy,
+    state.outOfOffice && ['busy', 'doNotDisturb', 'unknown'].includes(state.status) && styles.outOfOfficeBusy,
```

Comment:
Thank you for the fix! Would it be possible for 'unknown' to have a style in gray? 
![image](https://user-images.githubusercontent.com/13855169/156706130-8b85fc44-b2a1-4ecc-9667-7ad13a77ad5d.png)

Reply:
I don't think we can do that since that goes against our design spec. However, you can easily achieve that style customization yourself now that we added the status.